
geometry_msgs/PoseStamped goal

---

nav_msgs/Path plan
